builder/antlr/rules.bzl

```diff
@@ -0,0 +1,11 @@
+def python_grammar_adapter(
+        name,
+        input,
+        output):
```

Comment:
Can we make `output` optional? It doesn't need to be specified by the user most of the time. We can derive it using the input:

```
native.genrule(
  name = name,
  srcs = [input],
  outs = ['python_' + input]
  ...
)
```

Is my opinion correct @vmax ?

If that's the case, the only time a user needs to set the `output` is if they have multiple targets where each happens to generate a file with the same name. In that rare case, the user can override the default output to avoid the clash.

Reply:
@lolski it's actually `TypeQL<Language>.g4`, not `python_TypeQL`.

Thing is, we want users of the rule to be very intentional in how they name the output file - it affects how files, modules and classes are named in the generated code. If we generate the name for the user, we will lose the explicitness part.

Reply:
I see, got it @vmax .